docs/advanced-usage/.pages

```diff
@@ -4,3 +4,5 @@ arrange:
     - storage-backends
     - code-repositories
     - access-control
+    - containers
```

Comment:
*question:* why do you need to add `containers` here?

Reply:
Because it is missing from the list of pages, we probably forgot to add it when the containers page was written

Reply:
I was more curious why this change was there rather than the ordering of elements. Sorry for not being clear.

*suggestion:* It would be good to add it as a separate commit as this change is not part of user quotas docs.

But, if you already asked about ordering, usually we do alphabetic order in other places (code, etc.). I have checked some other docs pages and they are not in alphabetic order. But, I also don't think we should change the order here so just leave it as it is.